data-prepper-plugins/otel-trace-source/src/test/java/com/amazon/dataprepper/plugins/source/oteltrace/OTelTraceSourceTest.java

```diff
@@ -130,6 +131,18 @@
             .addResourceSpans(ResourceSpans.newBuilder()
                     .addInstrumentationLibrarySpans(InstrumentationLibrarySpans.newBuilder()
                             .addSpans(Span.newBuilder().setTraceState("FAILURE").build())).build()).build();
+
+    private static void assertStatusCode415AndNoServerHeaders(final AggregatedHttpResponse response, final Throwable throwable) {
```

Comment:
nit: the throwable is not used and can be removed from the method signature.

Reply:
I included the throwable to match the whenCompleted argument interface (BiConsumer<? super T, ? super Throwable>) to replace the inline lambda like this:
`.whenComplete(OTelTraceSourceTest::assertStatusCode415AndNoServerHeaders)`
I can remove the argument but it would pass the problem along to the wrapping lambda:
`.whenComplete((i, ex) -> assertStatusCode415AndNoServerHeaders(i))`

Reply:
Perhaps just change it to this:

```
@SuppressWarnings("unused")  final Throwable unused
```

Reply:
Another alternative would be to perform some validation on the throwable.

Reply:
Good idea @cmanning09, I'll enhance the test to assert the throwable is null.